### PR TITLE
Update docs on enable_reconfigure now that gcloud bug is fixed

### PR DESCRIPTION
**Pull request by nick-stroud**
Tested:
- Add partition
- Resize partition
- Add network storage
- Delete partition
- gcloud project config mismatch

### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md
Comment (mr0re1) [inline community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:93]: `--user` will throw if is run from virtualenv. Do we want to discourage usage of virtualenv?

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md
Comment (cboneti) [inline community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:81]: Optional: maybe add that  changing the VM type of a partition will likely not work. (I tried with a gpu vm today and it did not work -- as it fails to update the instance template). It seems like a common case.

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md
Comment (cboneti) [reply community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:93]: good point, also, it doesn't seem necessary if ran by non-root. It default to --user, right?

**Review by cboneti — APPROVED** [review]
I left two comments, consider it approved after you address them.

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md
Comment (nick-stroud) [reply community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:93]: I am reading that "pip defaults to installing Python packages to a system directory". I went through this process when testing and indeed the `--user` argument is denied when installing into virtualenv, but the error message is very clear. I would opt to leave it as we do not mention anything about a virtual environment. I also don't feel strongly so if you still disagree I can update.

Comment (nick-stroud) [reply community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:93]: In fact I just ended up removing `--user` as it does not make much difference to me and I want to move forward with this PR without waiting for re-review.

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md
Comment (nick-stroud) [reply community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:81]: Done.

**Pull request merged**
